docs/dev/makefile.rst

```diff
@@ -36,42 +36,61 @@ Python environment
 
    ``source ./local/py3/bin/activate``
 
-With Makefile we do no longer need to build up the virtualenv manually (as
-described in the :ref:`devquickstart` guide).  Jump into your git working tree
-and release a ``make pyenv``:
+With Makefile we do no longer need to build up the virtualenv manually. Jump
+into your git working tree and release a ``make pyenv``:
 
 .. code:: sh
 
    $ cd ~/searx-clone
    $ make pyenv
-   PYENV     usage: source ./local/py3/bin/activate
+   PYENV     [virtualenv] usage: source ./local/py3/bin/activate
+   PYENV     [virtualenv] installing requirements*.txt into ./local/py3
    ...
+   Successfully installed ...
 
 With target ``pyenv`` a development environment (aka virtualenv) was build up in
 ``./local/py3/``.  To make a *developer install* of searx (:origin:`setup.py`)
 into this environment, use make target ``install``:
 
-.. code:: sh
+.. code:: txt
```

Comment:
Why did you change it to `txt`? Isn't this block showing shell output?

Reply:
> Isn't this block showing shell output?

It shows the output! .. The (plain text) output of any command should not be highlighted by a bash syntax.  But thanks for the hint: it should be `text` not `txt` .. I fixed, rebased and force pushed ..